src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -201,44 +203,89 @@ def insar_tops_single_burst(
         packaging.upload_product_to_s3(product_dir, output_zip, bucket, bucket_prefix)
 
 
+def insar_tops_multi_burst(
+    reference: Iterable[str],
+    secondary: Iterable[str],
+    swaths: list = [1, 2, 3],
+    looks: str = '20x4',
+    apply_water_mask=False,
+    bucket: Optional[str] = None,
+    bucket_prefix: str = '',
+):
+    ref_unique_ids = [g.split('_')[1] + '_' + g.split('_')[2] + '_' + g.split('_')[4] for g in reference]
+    sec_unique_ids = [g.split('_')[1] + '_' + g.split('_')[2] + '_' + g.split('_')[4] for g in secondary]
+
+    if ref_unique_ids != sec_unique_ids:
+        raise ValueError('The reference burst(s) and secondary burst(s) do not match')
```

Comment:
A bit of a mouthful, but it's the only way I can think to make it clear what these variables actually represent.
```suggestion
    ref_number_swath_pol = [g.split('_')[1] + '_' + g.split('_')[2] + '_' + g.split('_')[4] for g in reference]
    sec_number_swath_pol = [g.split('_')[1] + '_' + g.split('_')[2] + '_' + g.split('_')[4] for g in secondary]

    if ref_number_swath_pol != sec_number_swath_pol:
        raise ValueError('The reference burst(s) and secondary burst(s) do not match')
```